src/Rings/mpoly.jl

```diff
@@ -205,8 +205,6 @@ singular_ring(::Nemo.FlintRationalField) = Singular.Rationals()
 singular_ring(F::Nemo.GaloisField) = Singular.Fp(Int(characteristic(F)))
 singular_ring(F::Nemo.NmodRing) = Singular.Fp(Int(characteristic(F)))
 
-singular_ring(R::Singular.PolyRing; keep_ordering::Bool = true) = R
```

Comment:
We need to keep this line.

Reply:
This must have been done by git. I did not touch this line. 
I will put it back.